internal/pkg/cli/env_init.go

```diff
@@ -798,7 +811,8 @@ func buildEnvInitCmd() *cobra.Command {
 	cmd.Flags().StringVar(&vars.tempCreds.SessionToken, sessionTokenFlag, "", sessionTokenFlagDescription)
 	cmd.Flags().StringVar(&vars.region, regionFlag, "", envRegionTokenFlagDescription)
 
-	cmd.Flags().BoolVar(&vars.isProduction, prodEnvFlag, false, prodEnvFlagDescription)
+	cmd.Flags().BoolVar(&vars.isProduction, prodEnvFlag, false, prodEnvFlagDescription) // Deprecated. Use telemetry flags instead.
```

Comment:
Do we need to update the description showing we are deprecating this flag?

Reply:
Not needed boss ✌️ because I've removed this flag at [L. 823](https://github.com/aws/copilot-cli/pull/3126/files/39fb9f9f27240e1b69130fb7a4070bd26d86c29b#diff-d61e7e024599d5b260984b80738cca7a2fc2a06ac37295b4c5900f9cea3fb8ebL823) so it doesn't show up in the help menu at all. However, it will still work for backwards compatible if a user is scripting the command